website/docs/r/data_factory_pipeline.html.markdown

```diff
@@ -41,16 +41,16 @@ resource "azurerm_data_factory_pipeline" "test" {
   }
   activities_json = <<JSON
 [
-	{
-		"name": "Append variable1",
-		"type": "AppendVariable",
-		"dependsOn": [],
-		"userProperties": [],
-		"typeProperties": {
-			"variableName": "bob",
-			"value": "something"
-		}
-	}
+ {
+  "name": "Append variable1",
+  "type": "AppendVariable",
+  "dependsOn": [],
+  "userProperties": [],
+  "typeProperties": {
+   "variableName": "bob",
+   "value": "something"
+  }
+ }
```

Comment:
we probably want to keep the tabs or replace them with enough spaces to match the formatting closer?

Reply:
The [original code](https://raw.githubusercontent.com/hashicorp/terraform-provider-azurerm/2daadb0f15ce7b64107c863ea76526b0b3c6f01a/website/docs/r/data_factory_pipeline.html.markdown) is using tab characters. Fixed the indentation with spaces with 3e8c1095f0ee184abbdaf6e0ea19cfdc55c15c78 commit. Is that okay?

Reply:
yep thanks!